config/locales/en.yml

```diff
@@ -230,6 +230,59 @@ en:
       link:
         label: "Offer tests or testing facilities (opens in a new window)"
       external_link: "https://www.gov.uk/guidance/help-the-government-increase-coronavirus-covid-19-testing-capacity"
+    check_your_answers:
+      charge: "How much you will charge"
```

Comment:
The `"` marks are redundant. YAML's default behaviour is to convert the entries to strings. Quotes are only needed if the text contains characters that the YAML parse may misinterpret (such as `:`). So:
```yaml
charge: How much you will charge
```

Reply:
Thanks Rob, yeah that makes total sense. That said, I think this should probably be a separate card, and be done across all three forms.